src/main/java/org/jenkinsci/backend/confluence/pageremover/webapp/PageRemoverResource.java

```diff
@@ -0,0 +1,180 @@
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jenkinsci.backend.confluence.pageremover.webapp;
+
+import com.cybozu.labs.langdetect.LangDetectException;
+import com.cybozu.labs.langdetect.Language;
+import com.google.common.base.Optional;
+import hudson.plugins.jira.soap.RemotePage;
+import org.apache.commons.mail.Email;
+import org.apache.commons.mail.EmailException;
+import org.apache.commons.mail.SimpleEmail;
+import org.jenkinsci.backend.confluence.pageremover.Connection;
+import org.jenkinsci.backend.confluence.pageremover.LanguageDetection;
+import org.jenkinsci.backend.confluence.pageremover.PageNotification;
+import org.jenkinsci.backend.confluence.pageremover.Space;
+import org.jenkinsci.backend.ldap.AccountServer;
+import org.jenkinsci.backend.ldap.Config;
+import org.kohsuke.stapler.config.ConfigurationLoader;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.io.File;
+
+import static org.apache.commons.lang.exception.ExceptionUtils.getFullStackTrace;
+import static org.jenkinsci.backend.confluence.pageremover.Spambot.BLACKLIST;
+
+@Path("/page-remover")
+@Produces(MediaType.TEXT_PLAIN)
+public class PageRemoverResource {
+    private final String space;
+    private final String mailRecipient;
+    private final String smtpServer;
+
+    public PageRemoverResource(String space, String mailRecipient, String smtpServer) {
+        this.space = space;
+        this.mailRecipient = mailRecipient;
+        this.smtpServer = smtpServer;
+    }
+
+    @POST
+    public Response submit(@QueryParam("subject") Optional<String> subject,
+                           @QueryParam("body-plain") Optional<String> body) throws Exception {
+
+        PageNotification pageNotification = PageNotification.parse(subject.or(""), body.or(""));
+
+        if (subject.or("").startsWith("Re: ") && body.isPresent()) {
+            removePage(subject.get(), body.get());
+            return Response.ok().build();
+        }
+
+        if (pageNotification != null) {
+            processNotification(pageNotification);
+            return Response.ok().build();
+        }
+
+        System.err.println("Not a confluence notification email: " + subject.or("[empty subject]"));
```

Comment:
dropwizard ships with logback, would you mind changing these things to use logback so we can configure some 4real logging for the bot